.travis.yml

```diff
@@ -26,8 +27,9 @@ services:
 
 env:
   global:
-    - HELM_URL=https://storage.googleapis.com/kubernetes-helm
-    - HELM_TGZ=helm-v2.9.1-linux-amd64.tar.gz
+    - DOCKER_HOST="tcp://86.119.32.161:443"
```

Comment:
Do we want to encrypt this?

Reply:
Open ports are secured:
```
ALLOW IPv4 443/tcp from 0.0.0.0/0
ALLOW IPv4 8443/tcp from 0.0.0.0/0
ALLOW IPv4 22/tcp from 0.0.0.0/0
```

- 443: Docker port, secured with TLS auth
- 8443: Kubernetes API port, secured with TLS auth
- 22: ssh, ssh-key only